ledger/src/account/mod.rs

```diff
@@ -1,3 +1,4 @@
+//! Repository for working with `Account` entities.
```

Comment:
This line shows up next to the module. I would just document this as `//! account module` or delete it as it is self explanatory.